File: Python/Algorithm/BeakJoon/conditional_statement/[2525] 오븐시계.py
H, M = map(int, input().split())
MakingTime = int(input())

hour = (M + MakingTime) // 60
min = (M + MakingTime) % 60

if M + MakingTime >= 60:
    if H + hour >= 24:
        H -= 24
    H = H + hour
    print(H, min)

else:
    if H > 24:
        H -= 24
    print(H, M + MakingTime)

File: C_language/Person/beakjoon/[2525]오븐시계.c
#include <stdio.h>

int main(void)
{
    // 변수 : 시간, 분
    int hour, min;
    // 변수 : 주어진 시간
    int givenmin;
    // 변수 : 합계 -> 시간을 분으로 전환하기 위한 작업
    int sum;

    scanf("%d %d", &hour, &min);
    scanf("%d", &givenmin);

    sum = hour * 60 + min + givenmin;

    printf("%d %d", sum / 60 % 24, sum % 60);
}